src/databases/OpenPMD/OpenPMDClasses/PMDFile.C

```diff
@@ -165,6 +175,16 @@ void PMDFile::ScanFileAttributes()
     // OpenPMD files always contain a data group at the root
     groupId = H5Gopen(fileId, "/",H5P_DEFAULT);
```

Comment:
Ok, it looks like `data` is a top-level group...at least in the one and only example file we have.

So, change line 165 to

```
    groupId = H5Gopen(fileId, "/data",H5P_DEFAULT);		
```

Reply:
Whoops! That won't work either. Sorry for confusion.

The code that follows here is looking for a number of attributes on the root group. When I examine our example, I see these attributes...

```
/                        Group
    Attribute: basePath scalar
        Type:      9-byte null-padded ASCII string
        Data:  "/data/%T/"
    Attribute: date scalar
        Type:      25-byte null-padded ASCII string
        Data:  "2018-02-06 09:52:07 -0800"
    Attribute: iterationEncoding scalar
        Type:      9-byte null-padded ASCII string
        Data:  "fileBased"
    Attribute: iterationFormat scalar
        Type:      9-byte null-padded ASCII string
        Data:  "data%T.h5"
    Attribute: meshesPath scalar
        Type:      7-byte null-padded ASCII string
        Data:  "fields/"
    Attribute: openPMD scalar
        Type:      5-byte null-padded ASCII string
        Data:  "1.1.0"
    Attribute: openPMDextension scalar
        Type:      native unsigned int
        Data:  1
    Attribute: particlesPath scalar
        Type:      10-byte null-padded ASCII string
        Data:  "particles/"
    Attribute: software scalar
        Type:      4-byte null-padded ASCII string
        Data:  "warp"
    Attribute: softwareVersion scalar
        Type:      1-byte null-padded ASCII string
        Data:  "4"
    Location:  1:96
```

So, I would say if it does NOT have a string-valued attribute named `"openPMD"` on the root group, then it is NOT an openPMD file. So, lets re-work this logic a bit

Reply:
FYI...I just examined this file on `develop` and don't see same logic there...

https://github.com/visit-dav/visit/blob/c074a9b97e2d6c9e0c360e2b1e8273cf4809babd/src/databases/OpenPMD/OpenPMDClasses/PMDFile.C#L315

What I see is that develop opens `/data` and iterates over attrs there and the one example file we have, all the attrs are on `/`, not `/data`